easybuild/easyconfigs/b/BBMap/BBMap-37.93-intel-2018a.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'MakeCp'
+
+name = 'BBMap'
+version = '37.93'
+
```

Comment:
no longer adding the java versionsuffix?

Reply:
No, since we're not doing that consistently, and the new check we added will ensure only one Java version is used as dep for `2018a` easyconfigs.